recipes/GeoTessCPP/LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2023, conda-forge
```

Comment:
This is the staged-recipes license. Please don't change it or use it as the package license. Please revert this change.

Reply:
Done, my apologies.  Misunderstood the docs.